src/Network/NWParameters.cs

```diff
@@ -183,6 +183,7 @@ public unsafe static NWParameters CreateUdp (Action<NWProtocolOptions> configure
 		unsafe static extern IntPtr nw_parameters_create_custom_ip (byte custom_ip_protocol_number, BlockLiteral *configure_ip);
 
 		[NoWatch, NoTV, NoiOS, Mac (10,15)]
+		[BindingImpl (BindingImplOptions.Optimizable)]
 		public unsafe static NWParameters CreateCustomIP (byte protocolNumber, Action<NWProtocolOptions> configureCustomIP = null)
```

Comment:
Is there a test for this method?

Reply:
it might not need to be a _complete_ test, i.e. the linker can only try to optimize it if it's used (otherwise it's just removed).

Reply:
My thoughts were that the implementation of this method is very different from most other optimizable methods, so it would be nice to have a test to make sure it works even after optimization.

Reply:
make sense, even if its been proven harder to get good network test :)

Reply:
+1 BUT I prefer to land that in a diff PR, you need to give me time to understand what https://developer.apple.com/documentation/network/3141919-nw_parameters_create_custom_ip?language=objc means with a 'a custom IP protocol.'